documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

```diff
@@ -3,38 +3,40 @@
 // // assembly-upgrade.adoc
 
 [id='assembly-upgrade-resources-{context}']
-= Strimzi resource upgrades
+= Strimzi custom resource upgrades
 
-The `v1alpha1` API version is deprecated for the following Strimzi resources:
+[role="_abstract"]
+After you have upgraded Strimzi to {ProductVersion}, you must ensure that your custom resources are using API version `{ApiVersion}`.
```

Comment:
I think we should make it more clear here that is should be done some time after installing `{ProductVersion}` and before installing the next minor version. Users have fairly lot of time for it, it does not need to be done immediately.

Reply:
Added: _You can do this any time after moving to {ProductVersion}, but the upgrades must be completed before the next minor version update._